### DISCUSSION TITLE
Current date in SwiftUI environment

### DISCUSSION THREAD

**Discussion by fonkadelic** [pointfreeco/isowords#93] [Q&A]
I've noticed that displaying of the time until tomorrow is implemented with a  custom environment property `date`.
Since this relies on the body evaluation of the SwiftUI view i'm not sure if this works as expected. 
For example if the app becomes inactive and and at a later point active again, SwiftUI usually doesn't evaluate the body with the consequence that the time will not be updated.  (Not investigated further but i think the `HomeView` could potentially be affected by this.)
Would be interesting to know why you decided to not using the view store here.

**Comment by mbrandonw** [comment]
What you are describing is correct, but when the app is foregrounded we fetch newest daily challenge results which causes the time to update in the UI.

What are you thinking the alternatives are?
